dojo-dungeon/src/starter/DojoStarter.java

```diff
@@ -1,16 +1,197 @@
 package starter;
 
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.audio.Music;
+import contrib.crafting.Crafting;
+import contrib.entities.EntityFactory;
+import contrib.level.generator.graphBased.RoomBasedLevelGenerator;
+import contrib.level.generator.graphBased.RoomGenerator;
+import contrib.level.generator.graphBased.levelGraph.Direction;
+import contrib.level.generator.graphBased.levelGraph.LevelGraph;
+import contrib.systems.*;
 import core.Game;
+import core.level.Tile;
+import core.level.TileLevel;
+import core.level.elements.ILevel;
+import core.level.utils.DesignLabel;
+import core.level.utils.LevelElement;
+import core.level.utils.LevelSize;
 import core.utils.components.path.SimpleIPath;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
+/** Starter for the dojo-dungeon game. */
 public class DojoStarter {
-  public static void main(String[] args) throws IOException {
-    Game.initBaseLogger();
-    Game.loadConfig(new SimpleIPath("dungeon_config.json"), KeyboardConfig.class);
-    Game.disableAudio(true);
+  private static final String BACKGROUND_MUSIC = "sounds/background.wav";
+
+  /**
+   * Start a new dojo-dungeon game.
+   *
+   * @param args the command line arguments, currently unused
+   */
+  public static void main(String[] args) {
+    try {
+      Game.initBaseLogger();
+      configGame();
+      onSetup();
+      Game.run();
+    } catch (Exception e) {
+      System.err.println(e.getMessage());
+      System.err.println("Exiting ...");
+    }
+  }
+
+  private static void createLevel() throws IOException {
+    // create a customised level comprising three nodes (rooms)
+    LevelGraph graph = new LevelGraph();
+    DojoRoom room1 = new DojoRoom(graph);
+    DojoRoom room2 = new DojoRoom(graph);
+    DojoRoom room3 = new DojoRoom(graph);
+
+    // connect the rooms
+    room1.connect(room2, Direction.SOUTH);
+    room2.connect(room1, Direction.NORTH);
+    room2.connect(room3, Direction.SOUTH);
+    room3.connect(room2, Direction.NORTH);
+
+    // link the rooms
+    room1.setNextRoom(room2);
+    room2.setNextRoom(room3);
+
+    // create the rooms in custom level
+    RoomGenerator gen = new RoomGenerator();
+    createRoom_1(gen, room1);
+    createRoom_2(gen, room2);
+    createRoom_3(gen, room3);
+
+    // remove trap doors, config doors
+    configDoors(room1);
+    configDoors(room2);
+    configDoors(room3);
+
+    // close the doors
+    room1.closeDoors();
+    room2.closeDoors();
+
+    // set room1 as start level
+    Game.currentLevel(room1.getLevelNode().level());
+  }
+
+  private static void configDoors(DojoRoom node) {
```

Comment:
@cagix Wäre es sinnvoll, diese Methode auch in die Raum-Klasse zu verschieben? Oder wäre dann zu viel/zu wenig in der Raum-Klasse? Bin mir nicht sicher, weil sich die Zuständigkeiten beider Klassen zum teil überschneiden.

Reply:
@tgrothe Du hast die Methode "angeschleppt" ;) - Du solltest doch am besten wissen, wo sie hingehört und gebraucht wird? 

Im Starter fand ich diese Methode als _API-Methode_ seltsam - der Starter wird ja von niemandem aufgerufen, d.h. was soll diese Methode dann in der öffentlichen API (aka `public`)? Als interne Hilfsmethode im Starter könnte sie vermutlich hilfreich sein (das musst Du einschätzen) - aber dann wäre sie aus meiner Sicht `private`. 

Wenn die Methode auf den Eigenschaften von `DojoRoom` arbeitet, dann klingt das für mich wie eine Methode dieser Klasse.

Reply:
Ok, dann würde es so, wie es jetzt ist (private und im Starter), für mich passen. Ein Raum muss nur einmal konfiguriert werden, zu Beginn.